lib/src/locator.dart

```diff
@@ -44,4 +46,12 @@ void setupLocator() {
       () => KanaRepository(),
       dependsOn: [Isar]);
   locator.registerSingleton<SettingsRepository>(SettingsRepository());
+
+  // Data Loaders
+  locator.registerSingletonWithDependencies<GroupDataLoader>(
+      () => GroupDataLoader(),
+      dependsOn: [Isar]);
```

Comment:
Simple question, est-ce que cela aurait pu etre ca a la place? 🤔 

```suggestion
  locator.registerSingletonAsync<GroupDataLoader>(
      () async {
          final instance = GroupDataLoader();
          await instance.loadCollection();
          return instance;
      },
      dependsOn: [Isar, ApiService]);
```

En soit, ce que tu as mis fonctionne très bien et en fonction de comment on va implementer la sync de données ça bougerait de là

Reply:
Si on fait ça là c'est pendant le splash screen d'android et pas le splash screen animé. (d'après ce dont on avait discuté ce weekend) Du coup c'est pour ça que je n'ai pas mis ça ici